packages/vsx-registry/src/browser/vsx-extension-commands.ts

```diff
@@ -36,6 +36,13 @@ export namespace VSXExtensionsCommands {
         label: nls.localizeByDefault('Install from VSIX') + '...',
         dialogLabel: nls.localizeByDefault('Install from VSIX')
     };
+    export const INSTALL_VSIX: Command = {
```

Comment:
I'm worried that the const name may make it easy to confuse this with `INSTALL_FROM_VSIX` but I understand it is hard to find a name without changing the existing ones. What do you think of `INSTALL_VSIX_FILE`?

Similar to the label, I'd prefer `Install VSIX extension` or just `Install Extension` with the `...` since it is a longer running operation but maybe the dots can be skipped (not sure if we only use them when a dialog opens).

Reply:
`INSTALL_VSIX_FILE` might be a better choice, I like it. 
Regarding the label, I tend to use the same as vs-code uses, so we can use the existing translation. 
And yes, `...` are normally for dialogs where an additional user action is required. Here it is not the case.

Reply:
If there is already a label for that in VS Code then let's go with that! I must have overlooked it, thank you for clarifying!